src/stores/PreferenceStore.ts

```diff
@@ -82,7 +84,7 @@ export enum PreferenceKeys {
     CATALOG_TABLE_SEPARATOR_POSITION = "catalogTableSeparatorPosition",
 
     PIXEL_GRID_VISIBLE = "pixelGridVisible",
-    PIXEL_GRID_COLOR = "pixelGridColor"
+    PIXEL_GRID_COLOR = "pixelGridColor",
```

Comment:
make sure your IDE's auto-formatting aligns with our `.prettierrc` file. We don't have trailing commas

Reply:
all set now, thanks.